jql-core/src/main/java/io/github/benas/jql/core/AnnotatedWithCalculator.java

```diff
@@ -47,11 +48,15 @@ public AnnotatedWithCalculator(TypeDao typeDao, AnnotatedWithDao annotatedWithDa
     public void calculate(ClassOrInterfaceDeclaration classOrInterfaceDeclaration, CompilationUnit compilationUnit) {
         List<AnnotationExpr> annotations = classOrInterfaceDeclaration.getAnnotations();
         for (AnnotationExpr annotation : annotations) {
-            String annotationName = annotation.getName().getName();
-            String annotationPackageName = ((CompilationUnit) annotation.getParentNode().getParentNode()).getPackage().getPackageName();
+            String annotationName = annotation.getNameAsString();
+            String annotationPackageName = annotation
+                    .findCompilationUnit()
+                    .flatMap(CompilationUnit::getPackageDeclaration)
+                    .flatMap(pkg -> Optional.of(pkg.getNameAsString())).orElse("???");
```

Comment:
Original code got a null when there is no `package` declaration. Not sure what to do here.

Reply:
if there is no package declaration, then the annotation is declared in the default package right?
What if we use an empty string as default value in the `orElse`?